pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -96,8 +96,22 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
                   <TableCell>Rhoncus aenean vel elit scelerisque mauris</TableCell>
                 </TableRow>
                 <TableRow>
-                  <TableHeader>UPL</TableHeader>
-                  <TableCell>Sit amet nisl suscipit adipiscing bibendum est ultricies</TableCell>
+                  <TableHeader>{t("Products")}</TableHeader>
+                  <TableCell>
+                    {_getComponent.data.embedded.nl.upl.map((product: string, idx: any) => (
```

Comment:
1. It's possible for `upl` to be unset. Might not be the case within the test data, but it could happen in the real data set. Therefore we cannot `.map()` on it without doing a check (mapping on `null` will break).

If `.embedded.nl.` is always available (which I think it should be, even though their values might be nullable, the path should be there)l. We could simply add an optional check, like so: `.nl?.upl.map( ... )`.

2. `idx` is of type `number`.